libyul/optimiser/Semantics.h

```diff
@@ -112,6 +112,20 @@ class MSizeFinder: public ASTWalker
 	bool m_msizeFound = false;
 };
 
+class PCFinder: public ASTWalker
```

Comment:
Where is this class called?

Reply:
dead code, i've been refactoring and not removing this one. thx.